.github/workflows/api-checks.yml

```diff
@@ -10,14 +10,11 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-# This Action runs checks things that we care about but would be
-# too slow to check in every PR.
-
-name: Extended checks
+name: API checks
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 0 * * *" # Every day at midnight UTC
+    - cron: "0 0 * * 0,2,4" # Every Sunday, Tuesday, and Thursday at midnight UTC
```

Comment:
Switched to 3 times a week to waste fewer resources. Most people aren't checking on Friday night and Saturday what happened, for example.